lib/src/components/navigation/NavigationMenuItem.tsx

```diff
@@ -71,7 +72,14 @@ const StyledLink = styled(NavigationMenuPrimitive.Link, itemStyles, {
       dropdownItem: {
         '&[data-active]': {
           background: '$primaryLight',
-          color: '$primary'
+          '*': {
+            color: '$primary'
+          },
+          '&:hover': { background: '$tonal50' },
+          '&:active': { background: '$tonal100' },
+          '&:focus-visible': {
+            boxShadow: `0 0 0 2px ${theme.colors.primary}`
```

Comment:
We can access the theme colours in every CSS key be prefixing with the "group" that they're in; in this case `$colors`. 

```suggestion
            boxShadow: `0 0 0 2px $colors$primary`
```

Reply:
Thanks, did not notice this